programs/ziti-edge-tunnel/instance.c

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Netfoundry, Inc.
+Copyright 2019 NetFoundry, Inc.
```

Comment:
it's not supposed to have the comma fwiw... we need to clean all these up everywhere still... :(

Reply:
CLion now has copyright update action!